contributors/devel/sig-instrumentation/migration-to-structured-logging.md

```diff
@@ -172,7 +172,7 @@ type KMetadata interface {
 1. Change log functions to structured equivalent
 1. Remove string formatting from log message
 1. Name arguments
-1. Use `klog.KObj` and `klog.KRef` for Kubernetes object references
+1. Use `klog.KObj` and `klog.KRef` for Kubernetes objects
```

Comment:
I think this is good as is, perhaps we should link to some API docs about object references?